lib/corm/model.rb

```diff
@@ -138,6 +140,43 @@ def self.count
       execute(statements['count']).first['count'].to_i
     end
 
+    ##
+    # Find by keys.
+    # This `find` methods wants to be as flexible as possible.
+    #
+    # Unless a block is given, it returns an `Enumerator`, otherwise it yields
+    # to the block an instance of the found Cassandra entries.
+    #
+    # If no keys as passed as parameter, the methods returns (an Enumerator for)
+    # all the results in the table.
+    #
+    # If one or more keys are given, the **order matters**; since this methods
+    # will craft a CSQL query, the first key **must** be the `partition key`,
+    # while the followings are intended as the `clustering keys`... that can be
+    # more than one.
+    #
+    # If the keys passed as parameter are more than the defined by the table the
+    # query is not valid, it cannot be executed and an error is raised.
+    def self.find(*key_values, &block)
```

Comment:
@zeroed I don't like the *key_values argument. I kindly suggest you to use an hash of keys/values because it's not clear from the code what is the order of the values in the array. You have to check primary_key() statement in the class definition, but the worst part is that the order is implicit in the implementation: it's really easy to mess things up with a refactoring (your code cannot cover how other users are using the lib).